gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Sep 09 23:13:46 BST 2013
+#Tue Jul 19 00:15:11 BST 2016
```

Comment:
Just wondering if this file was intentionally committed as it doesn't appear to be related to the changes in hand?